SUMMARY.md

```diff
@@ -1,24 +1,28 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/INSTALLATION.md)
+  * [Docker Installation](installation/INSTALLATION-DOCKER.md)
+  * [Local Installation](installation/INSTALLATION-LOCAL.md)
 * [Backend](backend/README.md)
+  * [Neo4j Database](neo4j/README.md)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-24T13:26:21Z" title="Wednesday, April 24th 2019, 3:26:21 pm +02:00">Apr 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,28 +1,34 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/README.md)
+  * [Docker Installation](installation/installation-docker.md)
+  * [Local Installation](installation/installation-local.md)
 * [Backend](backend/README.md)
+  * [Neo4j Database](backend/neo4j-database.md)
```


This whole section `neo4j` will be moved to a top level folder in https://github.com/Human-Connection/Human-Connection/pull/485: 
```suggestion
  * [Neo4j Database](neo4j/README.md)
```